src/renderer/widgets/Transfer/model/form-model.ts

```diff
@@ -237,11 +238,20 @@ const $fakeTx = combine(
 
     const transactionType = network.asset.type ? TransferType[network.asset.type] : TransactionType.TRANSFER;
 
+    const palletName =
```

Comment:
I think it's time to move all duplicates into separate method somewhere. Maybe 'chainsService' for now